src/connection.ts

```diff
@@ -136,7 +137,8 @@ export async function importConnFromDbfacts() {
     connId = '(dbfacts)' + connId
     const params = parseDbFactsConnection(connId)
     const connection = {
-        'type': params['type']
+        'type': params['type'],
+        'ssl': await sslPicker()
```

Comment:
Use sslPicker when importing connection from db-facts. by default db-facts does not have an `ssl` parameter